### ISSUE TITLE
/bin/python: symbol lookup error: ../site-packages/torch/lib/libtorch_python.so: undefined symbol: PySlice_Unpack

### ISSUE THREAD

**Issue by mengjiexu** [fastai/fastai#1298]
when I use :
DataLoader(MyDataSet(train_csv, train_dir, 'test',
                                        transforms_test, input_shape),
                              batch_size=BATCH_SIZE,shuffle=True, num_workers=0, pin_memory=False)
to load my data, It works.
when I use above to train and valid in：
Learner(databunch, model, loss_func=nn.CrossEntropyLoss(), metrics=[accuracy, topkaccuracy])
It works fine when use train data, but dead kernel when start forward with valid data.

**Comment by samarthaggarwal** [comment]
Since you have closed the issue, it seems that you have found the resolve. Could you please share the resolve. I am facing the same error.

**Comment by mengjiexu** [comment]
@samarthaggarwal reboot your computer, the only solution that I have found(-_-).

**Comment by samarthaggarwal** [comment]
@mengjiexu thanks for your advice. I am actually using gpu on a shared server. I cannot reboot the system. As far as logging out and logging in again is concerned, I have tried that. It did not work. Any other suggestions? Or any idea that you think could be a potential resolve?

**Comment by mengjiexu** [comment]
reinstall the fastai and pytorch

**Comment by samarthaggarwal** [comment]
I sure i'll try that. Also, I had installed pytorch globally in my system and not inside any conda environment. Do you think that can be the issue?

**Comment by mengjiexu** [comment]
Maybe. It should not be installed globally.
